test/lib/find-scaffolds.js

```diff
@@ -58,22 +63,52 @@ const after = () => {
 };
 
 
-test('Find Scaffolds success', assert => {
+test('Find Scaffolds API', assert => {
+  before();
```

Comment:
:+1: On a related note... I'm going into lots of detail about how to write better tests in a [webcast next week](https://ericelliottjs.com/product/lifetime-access-pass/).